### DISCUSSION TITLE
Allow for different 2FA Methods i.e. yubikey

### DISCUSSION THREAD

**Discussion by grafst** [Jacobtims/filament-breezy#186] [Ideas]
It would be very cool to be able to use my yubikey as a second factor.
Maybe with this package: https://github.com/bitbeans/Yubikey?

**Comment by ryanshipswell** [comment]
I second this strongly. It would be great to offer a variety of 2FA methods and even rules for which are offered to which users. So I could say me and other admins must use stronger 2FA methods, whereas a regular user could get more basic or even no 2FA on their account.

I use my Yubikey wherever possible, I especially want to be able to use it on my own apps using this because I generally have full admin access to literally everything so I of course want the strongest security on my own account.
